### PR TITLE
docs: Modifying snapshot

### PR DESCRIPTION
**Pull request by chivay**

### CODE REVIEW

File: docs/usage/managing_snapshots.rst [hunk 1]
```diff
@@ -2,6 +2,66 @@
...
+.. code-block:: ipython
+
+    In [10]: exit()
+
+or hit Ctrl+D.
+
+.. warning::
+    vm-0 is a base for other virtual machines. Leaving it in a broken or inconsistent state will
+    result in analysis failures, BSODs and other unexpected errors. When modifying the vm-0 always
+    make sure to perform the postinstall step.
```
Comment (BonusPlay) [inline docs/usage/managing_snapshots.rst:59]: This section is unclear.

Comment (BonusPlay) [inline docs/usage/managing_snapshots.rst:59]: This section is unclear.

Comment (chivay) [reply docs/usage/managing_snapshots.rst:59]: What seems to be unclear?

Comment (BonusPlay) [reply docs/usage/managing_snapshots.rst:59]: > make sure to perform the postinstall step.

what postinstall step?

Comment (chivay) [reply docs/usage/managing_snapshots.rst:59]: The one mentioned on line 45 :thinking:

Comment (BonusPlay) [reply docs/usage/managing_snapshots.rst:59]: OK, missed that one.

**Review by BonusPlay — APPROVED** [review]

**Pull request merged**